### PR TITLE
Use model errors helper in devise views

### PR DESCRIPTION
**Pull request by RobertCram**
Story: https://www.pivotaltracker.com/story/show/149263775

**Changes proposed in this pull request:**

Use the model_errors_helper in the devise password and registration views and bring view layouts in line with other views.

password

Ready for review:
@patmbolger @weedySeaDragon

### CODE REVIEW

**Review by weedySeaDragon — APPROVED** [review]

File: app/views/devise/passwords/edit.html.haml [hunk 1]
```diff
@@ -4,20 +4,17 @@
   %span
 .entry-content.wpcf7
   = form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :put }) do |f|
-    - if resource.errors.any?
-      .wpcf7-response-output
-        - resource.errors.full_messages.each do |msg|
-          = msg
+
+    != model_errors_helper(resource)
+    %br/
```
Comment (patmbolger) [inline app/views/devise/passwords/edit.html.haml]: I like the break between the errors div and the next section of the form.   How about adding that to the helper method as part of this PR?

**Review by patmbolger — COMMENTED** [review]
Please see comment.

File: app/views/devise/passwords/edit.html.haml (hunk 1 above)
Comment (RobertCram) [reply app/views/devise/passwords/edit.html.haml]: Done. See below for an example of this change in other forms. 

An unrelated note: The the text at the very top is not styled, and not seperated nicely from the text following it. Is this intentional or should we enter a bug?

membershipapplication

Comment (patmbolger) [reply app/views/devise/passwords/edit.html.haml]: Yes, please enter a bug.  We could add some margin to the top-most div containing the error text in order to separate the error section from the text above it.

**Review by patmbolger — APPROVED** [review]

**Pull request merged**